client/lib/media/actions.js

```diff
@@ -37,7 +37,11 @@ MediaActions.setQuery = function( siteId, query ) {
 	} );
 };
 
-MediaActions.fetch = function( siteId, itemId ) {
+// TODO: Once this entire module is converted to Redux,
+// `receiveMediaItems` shouldn't be passed as an argument anymore,
+// but directly imported and dispatch()ed from inside `fetch()`,
+// which needs to be turned into a Redux thunk.
+MediaActions.fetch = function( siteId, itemId, receiveMediaItems = null ) {
```

Comment:
It's true #2167 would make it easy for us. That's a shame, because we've been discussing the argument approach with @ockham since the GM 😞  But if the best we can do right now is to punt this PR until we get #2167 merged, I'd agree to do so. Do you folks think punting is the right way to go, @aduth @ockham?